src/durations.config.ts

```diff
@@ -0,0 +1,45 @@
+export type DurationConfig = Record<string, number>
+
+const durationsMillis: DurationConfig = {
```

Comment:
This would be better typed `as const` so that we can extract the keys as compile time.

Reply:
Good call! This is not a TypeScript feature I've used before.